### PR TITLE
Release/v1.3.0

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,12 +24,13 @@
...
   "homepage": "https://github.com/3box/3box-api#readme",
   "dependencies": {
-    "3box-orbitdb-plugins": "^1.0.6",
+    "3box-orbitdb-plugins": "3box/3box-orbitdb-plugins#feat/orbit-23",
     "3id-resolver": "^0.0.4",
     "analytics-node": "^3.3.0",
     "aws-sdk": "2.414.0",
     "axios": "^0.19.0",
-    "cids": "0.7.1",
+    "bunyan": "^1.8.12",
```
Comment (oed) [inline package.json]: What's this?

Comment (zachferland) [reply package.json]: its like the third time you ask ha :) its logging tooling, not used directly, but missing in another dependency (express-bunyan-logger)

Comment (oed) [reply package.json]: haha, ok. I just noticed that it wasn't used anywhere :P

Comment (msterle) [reply package.json]: might be able to remove this is an explicit dep - I forget why I added it explicitly. Not important to do in this PR though

**Pull request merged**